playwright/e2e-vrt/scenes-app/insights.spec.ts

```diff
@@ -132,12 +132,12 @@ test.describe('stickiness insight', () => {
     })
 })
 
-test.describe('user paths insights', () => {
-    test('displays viz correctly', async ({ storyPage }) => {
-        await storyPage.goto(toId('Scenes-App/Insights', 'User Paths'))
-        await storyPage.expectSceneScreenshot()
-    })
-})
+// test.describe('user paths insights', () => {
```

Comment:
should probably `test.skip` rather than comment it. and add an explanation comment for the future traveller (even just pointing to this PR)